heron/schedulers/src/java/com/twitter/heron/scheduler/nomad/NomadScheduler.java

```diff
@@ -356,12 +358,12 @@ static void startJobs(NomadApiClient apiClient, Job... jobs) {
       LOG.fine("Starting job " + job.getId());
       try {
         EvaluationResponse response = apiClient.getJobsApi().register(job);
-        LOG.info("response: " + response);
+        LOG.fine("response: " + response);
       } catch (IOException | NomadException e) {
         String msg = "Failed to submit job "
-            + job.getId() + " with error: " + e.getMessage()
-            + " with job spec: " + job.toString();
-        LOG.log(Level.SEVERE, msg, e);
+            + job.getId() + " with error: " + e.getMessage();
+
+        LOG.log(Level.FINE, msg + " with job spec: " + job.toString());
```

Comment:
1. Why change to FINE? This error looks serious.
2. why append strings in two places? One in msg =, another in LOG.log()?

Reply:
To answer you first question:

The job spec can be quite long that is why I would rather print it explicitly if the user specified --verbose via the command line.

To answer your second question:

The reason why I append if so that the RuntimeException thrown only contains the error message and not the whole job spec but I would log the error message and the job spec if verbose is set